client/src/VideoPlayerLarge.jsx

```diff
@@ -62,9 +63,15 @@ class VideoPlayerLarge extends React.Component {
       <div className="videoPlayerLarge">
       <div className="large-player">
         <i onClick={() => {this.props.toggleLargeVideo()}} className="exit-large-player fas fa-times"></i>
-        <video onClick={() => {this.playPause()}} autoPlay onEnded={() => {this.setState({playing: false, paused: true})}} onTimeUpdate={() => {this.setState({currentTime: document.getElementById('videoToPlay').currentTime})}} id="videoToPlay" key={this.props.selectedIdx}>
+        <video onMouseOver={() => {this.setState({showNext: true})}} onMouseOut={() => {this.setState({showNext: false})}} onClick={() => {this.playPause()}} autoPlay onEnded={() => {this.setState({playing: false, paused: true})}} onTimeUpdate={() => {this.setState({currentTime: document.getElementById('videoToPlay').currentTime})}} id="videoToPlay" key={this.props.selectedIdx}>
```

Comment:
Instead of writing your mouseOver function within the element itself, you could define it outside of render, that way if you want to use it in multiple places (like it looks like you did on line 70 for mouseEnter), you only have to define it once, it will help your code look a little cleaner, and it will make it easier on you to refactor in the future, if needed.